### ISSUE TITLE
Text cursor is always mispositioned from mouse click

### ISSUE THREAD

**Issue by lwr** [atom/atom#6152]
![7c36e257-286c-41b3-95d8-dc9014453f84](https://cloud.githubusercontent.com/assets/793171/6884318/788f450a-d61e-11e4-9c39-e7c0353f01d3.png)

Just as the picture above.
This annoying problem has been come up for a very long time.

I have tried all these and the problem is the same
- from first beta release till now 0.188.0
- from OS X 10.8.4 through 10.9 till now 10.10.2
- using --safe mode
- try different theme
- try different font families / sizes / line-heights

The problem is only found in editor, it can not reproduced in any other interface, etc. settings tab

I am very confusing if I could diagnose on this problem by any other way?

**Comment by YuriSolovyov** [comment]
+1, very annoying

**Comment by izuzak** [comment]
@lwr I'm not sure I understand the report or how to reproduce this. Can you share a GIF which clearly demonstrates the problem and provide step-by-step instructions to reproduce? For me, the cursor always shows up where I clicked.

**Comment by YuriSolovyov** [comment]
@izuzak when I took a closer look at this, I think I +1-ed too early, though there is a problem where if you leave cursor at some place, scroll a couple of times and then click somewhere, it scrolls in some wierd position, even if I clicked in center of screen and there is no need to scroll.

I do not seeing anything like @lwr, but issue title almost fits, with correction that scroll position is misplaced, not cursor.

I may fill another issue if needed.

**Comment by lwr** [comment]
![1](https://cloud.githubusercontent.com/assets/793171/6900283/07669c0a-d738-11e4-8b42-20890c5a7bd5.gif)
@izuzak @YuriSolovyov 
I have toke a GIF, hope it is clear enough.

I have no idea how it could be happened, and search for similar issue but got no report.

**Comment by as-cii** [comment]
@lwr: could you report here which font family are you using, please? It'd be great if you could paste your entire `config.cson` :bow: 

@izuzak: I suspect it's an issue related to how we measure characters, but let's see if we can reproduce this first :ok_hand:

**Comment by lwr** [comment]
```json
cat ~/.atom/config.cson
"*":
  "exception-reporting":
    userId: "2703a42e-3aee-3c97-ecd2-26c571fef127"
  "release-notes":
    viewedVersion: "0.95.0"
  welcome:
    showOnStartup: false
  metrics:
    userId: "34c412b4fdd68fe03620515b78577212ed0f3f04"
  editor:
    preferredLineLength: 120
    showInvisibles: true
    invisibles: {}
    fontFamily: "InputMonoCompressed-ExtraLight, STXihei"
  core:
    themes: [
      "atom-light-ui"
      "atom-light-syntax"
    ]
    disabledPackages: [
      "metrics"
      "vim-mode"
    ]
  "markdown-preview":
    breakOnSingleNewline: true
```

```css
 cat ~/.atom/styles.less
/*
 * Your Stylesheet
 *
 * This stylesheet is loaded when Atom starts up and is reloaded automatically
 * when it is changed.
 *
 * If you are unfamiliar with LESS, you can read more about it here:
 * http://www.lesscss.org
 */

.tree-view {

}

.editor {
    zoom:75%;
}

.editor .cursor {

}

.markdown-preview {

  font-family: "HelveticaNeue-Light", sans-serif;

  pre, code, pre code {
    font-family: "Ubuntu Mono", "STXihei", /* "M+ 1m", */monospace;
  }

  code, th, td {
    white-space: nowrap;
  }

  pre, pre code {
    white-space: pre;
    line-height: 1.3em;
  }

  code {
    padding: 2px 5px;
  }
}
```

@as-cii 
thank you for your hints. I had just found the problem by myself.
That is caused by css `.editor { zoom }`
I just don't remember when I had add this to style.less, maybe for years 😳
Now it is fixed by removeing this line.

**Comment by 50Wliu** [comment]
Nice to see that you found your issue!

**Comment by 50Wliu** [comment]
Also @YuriSolovyov your issue should be fixed in the next Atom version, 0.189.0.

**Comment by YuriSolovyov** [comment]
Thanks, will try
